torch/_torch_docs.py

```diff
@@ -1903,6 +1903,55 @@ def merge_dicts(*dicts):
     tensor([-1 - 1j, -2 - 2j, 3 + 3j])
 """.format(**common_args))
 
+add_docstr(torch.copysign,
+           r"""
+copysign(input, other, *, out=None) -> Tensor
+
+Returns a new tensor with the magnitude of :attr:`input` and the sign of :attr:`other` elementwise.
+
+.. math::
+    \text{out}_{i} = \begin{cases}
```

Comment:
```
$$
\text{out}_{i} = \begin{cases}
        -|\text{input}_{i}| & \text{if } \text{other}_{i} < 0 \:(\text{or}\:\text{other}_{i} == -0.0) \\
        |\text{input}_{i}| & \text{if } \text{other}_{i} \geq 0\\
    \end{cases}
$$
```

This will space the parenthetical clause in the first case and removes it in the second case by using geq.